backend_new/src/dtos/reserved-community-types/reserved-community-type-query-params.dto.ts

```diff
@@ -0,0 +1,14 @@
+import { Expose } from 'class-transformer';
+import { ApiProperty } from '@nestjs/swagger';
+import { IsString } from 'class-validator';
+import { ValidationsGroupsEnum } from '../../enums/shared/validation-groups-enum';
+
+export class ReservedCommunityTypeQueryParams {
+  @Expose()
+  @ApiProperty({
+    required: false,
+    type: String,
+  })
+  @IsString({ groups: [ValidationsGroupsEnum.default] })
+  jurisdictionName?: string;
```

Comment:
Is this consistent with what we're doing with ami charts? I think we're moving that to use the id? It looks like you may not have brought that merge in here yet, so flagging for the update.

Reply:
good call out, looks like we don't use either so switching to jurisdiction id!